<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateACustomDictionaryWindow</class>
 <widget class="QMainWindow" name="CreateACustomDictionaryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>643</width>
    <height>236</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="CreateACustomDictionaryCentralWidget">
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>643</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(255, 255, 255);
	border: none;
	border-radius: 7px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="blankForTheNameOfDictionary">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>55</y>
       <width>561</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>13</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color: rgb(255, 255, 255);
	border: 1px solid rgb(229, 229, 229);
	border-radius: 7px;
	padding: 12px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Name of the custom dictionary</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: none;
	border: none;
}</string>
     </property>
     <property name="text">
      <string>Name of the custom dictionary</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>320</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(214, 214, 214);
	border-radius: 7px;
	color: rgb(255, 255, 255);
	border: none;
}

QPushButton:hover {
	background-color: rgb(208, 208, 208);
}

QPushButton:pressed {
	background-color: rgb(202, 202, 202);
}</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveAndCreateADictionaryButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>320</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(56, 236, 113);
	border-radius: 7px;
	color: rgb(255, 255, 255);
	border: none;
}

QPushButton:hover {
	background-color: rgb(69, 241, 124);
}

QPushButton:pressed {
	background-color: rgb(48, 221, 103);
}</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="blankForTheDescription">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>164</y>
       <width>561</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>13</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit {
	background-color: rgb(255, 255, 255);
	border: 1px solid rgb(229, 229, 229);
	border-radius: 7px;
	padding: 12px;
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="overwriteMode">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Enter the description</string>
     </property>
    </widget>
    <widget class="QLabel" name="descriptionLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: none;
	border: none;
}</string>
     </property>
     <property name="text">
      <string>Descritpion</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
